client/src/actions/team.js

```diff
@@ -16,24 +16,16 @@ import {
   DEPLOY_FAIL,
   UPDATE_TEAM,
 } from './types';
-import { errorActionGlobalCreator } from '../error/errorReduxFunctions';
+import { errorActionGlobalCreator, successMessageCreator } from '../notification/notificationReduxFunctions';
 
 /**
- * Adds a new team to redux store and database.
+ * Create a new team to database.
  * @param teamInfo contains teamName, orgName and email
  */
 export const addTeamInfo = (teamInfo) => async (dispatch) => {
   try{
-    const teamId = await api.addTeam(teamInfo);
-    const teamData = {
-      ...teamInfo,
-      teamId: teamId,
-    };
-    // TODO: do we need to dispatch this action? 
-    dispatch({
-      type: ADD_TEAM,
-      payload: teamData,
-    });
+    const successMessage = await api.addTeam(teamInfo)
```

Comment:
As with my comment on the API, does renaming this function to `createTeam` make it more meaningful? Also, the action creator should be renamed accordingly as well (i.e. `addTeamInfo`)